tests/dec/test_dec.py

```diff
@@ -411,6 +411,13 @@ def test_master_DECAYdotDEC_file():
 
     assert p.number_of_decays == 506
 
+def test_BELLE2_decfile():
+    p = DecFileParser(DIR / '../../decaylanguage/data/DECAY_BELLE2.DEC')
+    p.parse()
+
+    # Just check the dec file will parse since I do not know 
+    # how many decays are in the dec file.
+    assert p.number_of_decays > 0
```

Comment:
Can't you run this once, get the number, then set it here? Then if we fix/improve/update the file, it will change and need to be changed here too, but at least we will know when it changes.

Reply:
I made the same comment above. I do check the correct number in a similar test of the LHCb master file.